src/Controls/tests/DeviceTests/Elements/HybridWebView/HybridWebViewTests.cs

```diff
@@ -7,7 +10,7 @@
 namespace Microsoft.Maui.DeviceTests
 {
 	[Category(TestCategory.HybridWebView)]
-	public class HybridWebViewTests : ControlsHandlerTestBase
+	public partial class HybridWebViewTests : ControlsHandlerTestBase
```

Comment:
possibly nit, but is there still a reason to make this a partial class?

Reply:
Not a nit at all, but yes there is a reason: the JS context nested type at the bottom of the file is also a partial class, so that the Code Analyzer can 'add' its compile-time-generated code. And for that nested type to be partial, so must the containing type - which is this test type.

Reply:
Ah makes sense!